handler/implementations/proc.go

```diff
@@ -232,31 +231,13 @@ func (h *Proc) readSwaps(
 		return 0, io.EOF
 	}
 
-	name := n.Name()
-	path := n.Path()
-	cntr := req.Container
-
-	// If no modification has been ever made to this container's swapping mode,
-	// then let's assume that swapping in OFF by default.
-	data, ok := cntr.Data(path, name)
-	if !ok || data == "swapoff" {
-		result := []byte(swapsHeader + "\n")
-		return copyResultBuffer(req.Data, result)
-	}
-
-	var result []byte
+	// Pretend swapping is off
+	//
+	// TODO: fix this once Sysbox intercepts the swapon() and swapoff() syscalls.
```

Comment:
The previous code was dead code: it will always return `req.Data = []byte(swapsHeader + "\n")`. 

The reason: there is a readSwaps(), but no writeSwaps() function that would modify the cntr.Data() cache entry for `/proc/swaps`.  As a result, line 242 in the prior code would always be true and return `[]byte(swapsHeader + "\n")`.

Reply:
Question: should there be a writeSwaps() function? If the answer is yes, then that's a different issue we need to fix (not in this change).

Reply:
`/proc/swaps` is a read-only resource, that's why we don't have a write() implementation. The kernel does the 'write' through swapon() syscall, which we need to intercept to complete this feature (seccomp-tracer would write into the sys-container's data map). Once we do that (low-hanging-fruit apparently), we'll need this code that you are eliminating, so i would leave it as it's not hurting us.

Reply:
As discussed, let's delete the code until we actually implement the functionality (otherwise it's confusing).